util/predicates/cluster_predicates_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package predicates_test
+
+import (
+	"testing"
+
+	"github.com/go-logr/logr"
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/cluster-api/util/conditions"
+	"sigs.k8s.io/cluster-api/util/predicates"
+)
+
+func TestClusterControlplaneInitializedPredicate(t *testing.T) {
```

Comment:
The cases these tests cover look. good. 👍🏼 
Can you please refactor the tests to follow the pattern used in other places in the project? Ref for example: https://github.com/kubernetes-sigs/cluster-api/blob/7acabeedc1c8f61d64cf97bbfea5247b9a73ce0a/util/util_test.go#L44-L109

This way the output of the tests becomes more readable in case of test failures and also the tests become easier to maintain.